feature-libs/checkout/base/components/checkout-delivery-mode/checkout-delivery-mode.component.ts

```diff
@@ -100,6 +101,33 @@ export class CheckoutDeliveryModeComponent {
     protected activeCartFacade: ActiveCartFacade
   ) {}
 
+  onKeydown(code: string | undefined, event: Event): void {
```

Comment:
I think that we should unit and e2e test this. E2E just to cover this is working in the user's environment.

Reply:
I've provided a unit and e2e test.